cmdrs-guide.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv="X-UA-Compatible" content="chrome=1">
+    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
+    <link rel="stylesheet" type="text/css" href="stylesheets/github-dark.css" media="screen">
+
+    <title>E:D Community Developers - CMDRs Guide</title>
```

Comment:
Since 2016 or so `Elite: Dangerous` changed to `Elite Dangerous`. Unrelated to PR since you use the correct style where it is used, but EDCD should change the name to be without colon

Reply:
Yep. I kept the current style but I agree with you.

Reply:
I'm all for a global E:D -> ED replace.  The only reason I've not done it in EDMC text is related to translations and our documentation also needing updating to match.